plugin.js

```diff
@@ -46,6 +46,44 @@ const add = async function (context) {
     print.info(MANUAL_INSTALL_INFO)
   }
 
+  // Patch for animatable
+  const pluginExamplesScreen = `${process.cwd()}/ignite/DevScreens/PluginExamplesScreen.js`
+  try {
+    const animatableExample = `${process.cwd()}/node_modules/ignite-animatable`
+    if (!filesystem.exists(animatableExample)) {
+      ignite.patchInFile(pluginExamplesScreen, {
+        replace: 'import \'../Examples/Components/animatableExample.js\'',
+        insert: '// animatableExample removed - ignite-animatable not installed [DO NOT REMOVE THIS LINE!]'
```

Comment:
We probably don't need the `DO NOT REMOVE THIS LINE` part, right?

Reply:
We don't, but if they remove it, then install `ignite-animatable`, then if they run `ignite add ignite-dev-screens`, the `import` line won't be restored. (Though I realize I need to make the line in the `else` block match this. I'll go ahead and remove it though.

Reply:
We can probably fix that without the comment. Just have it import below something else.